synchers/syncutils.go

```
@@ -251,7 +251,7 @@ func SyncCleanUp(environment Environment, syncer Syncer, dryRun bool, sshOptions
 	}
 
 	transferResourceName := transferResouce.Name
-	execString := fmt.Sprintf("rm -r %s", transferResourceName)
+	execString := fmt.Sprintf("rm -r %s || true", transferResourceName)
```

Why is this change required @timclifford? Is it because of the .tar.gz we're adding?

---

I noticed this when testing the file name change for the db name (appending .gz to the end). The clean up step was failing if the file could not be found - this just ignores the error if that happens. It shouldn't actually crop up again, but no harm in keeping it since its just a clean up step